### DISCUSSION TITLE
Listbox: Native on mobile as an option

### DISCUSSION THREAD

**Discussion by diar-kourehpaz** [tailwindlabs/headlessui#443] [Ideas]
I think it's great that we get to style our own selects that offer native functionality, but I think it would be better to let the respective platforms (iOS & Android) take care of it because the experience is so much better. I understand if not everyone agrees with this sentiment and that's why it should be an option.

Of course this already can be done having both Listbox and the native select, hiding one of them on different screen sizes. But since headlessui is all about the great experience it offers out of the box, maybe this feature request could be considered.

**Comment by theskillwithin** [comment]
Ya I would like to implement this on mobile
